### PR TITLE
WordBazaar No Raw Text and Page Titles

### LINKED ISSUE #961: no raw text

### LINKED ISSUE #962: page titles

### PR DESCRIPTION
**Pull request by AtanasKrondev**
Closes #961
Closes #962

### CODE REVIEW

File: src/views/WordBazaar.vue [hunk 1]
```diff
@@ -103,6 +102,7 @@ export default {
       this.wordRegistryState = await Backend.getWordRegistry();
     },
   },
+  metaInfo: { title: 'WordBazaar' },
```
Comment (mradkov) [inline src/views/WordBazaar.vue:105]: Shouldn't this also be part of the locales file and used as i18n token?

File: src/views/WordDetail.vue [hunk 2]
```diff
@@ -490,6 +500,7 @@ export default {
       }
     },
   },
+  metaInfo() { return { title: `${this.selectedWord}: Word Details` }; },
```
Comment (mradkov) [inline src/views/WordDetail.vue:503]: Same as above.

File: src/views/WordBazaar.vue (hunk 1 above)
Comment (AtanasKrondev) [reply src/views/WordBazaar.vue]: Indeeed

File: src/locales/en.json [hunk 3]
```diff
@@ -542,7 +542,54 @@
...
+      "Buy": "Buy",
+      "Sell": "Sell",
+      "AccountBalance": "Account Balance:",
+      "AmountBuying": "Amount buying:",
+      "AmountSelling": "Amount selling:",
+      "TotalPay": "Total you pay:",
+      "TotalGet": "Total you get:"
+    },
+    "WordDetail": {
+      "Title": "{word}: Word Detailsoou",
```
Comment (davidyuk) [inline src/locales/en.json:562]:
```suggestion
      "Title": "{word}: Word Details",
```
?

**Review by davidyuk — APPROVED** [review]

**Pull request merged**